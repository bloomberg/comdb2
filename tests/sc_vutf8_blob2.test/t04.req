insert into t4 values(1, 'CDB2BABE', x'CDB2BABE')
select * from t4
alter table t4 { tag ondisk { int i int j null=yes vutf8 txt[4] blob binary[4] } }$$
select * from t4
