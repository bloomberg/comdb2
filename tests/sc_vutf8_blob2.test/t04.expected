(rows inserted=1)
(i=1, txt='CDB2BABE', binary=x'cdb2babe')
(i=1, j=NULL, txt='CDB2BABE', binary=x'cdb2babe')
